api/envoy/config/route/v3/route_components.proto

```diff
@@ -457,6 +457,28 @@ message RouteMatch {
     // Note that CONNECT support is currently considered alpha in Envoy.
     // [#comment:TODO(htuch): Replace the above comment with an alpha tag.
     ConnectMatcher connect_matcher = 12;
+
+    // Supports segment based url_template matching for *:path" header.
+    // This match is always case sensitive.
```

Comment:
Do we want to force this or support case normalization? Seems we could do that.

Reply:
the path_matcher library doesn't support case insensitive. It requires some work to support it.  We can start it without it.

Reply:
I guess I'm asking if we can force lower case the matcher and input string on the Envoy-side?

Reply:
BTW, I don't think this is a super important ask, just curious if there is anything preventing a reasonable Envoy-side handlling.

Reply:
We could do that: for case insensitive, envoy side has to lower_case both url_template and path.   lower_case path for each request may have performance hit.